### ISSUE TITLE
train my own datesets

### ISSUE THREAD

**Issue by kungkook** [giovanniguidi/deeplabV3-PyTorch#1]
Can you train your own data set, such as camvid data set, or voc data set

**Comment by giovanniguidi** [comment]
Of course, you just need to change the data generator

**Comment by kungkook** [comment]
> Of course, you just need to change the data generator

The image of the training set must be in JPG format and the label in PNG format? Is there any requirement on the size of the image of the data set? Thank you for your answer

**Comment by kungkook** [comment]
> 当然，您只需要更改数据生成器

What version of pytorch is the running environment? In addition, I have seen deepfashion. Py. Could you please tell me how to generate this file for the data set I made by myself? Looking forward to your reply

**Comment by giovanniguidi** [comment]
Take a look at deepfashion.py and the README, it's all there

**Comment by kungkook** [comment]
Hello, I was trying to run the code under ubuntu and 'torch encoding 1.0.1 required nose, which is not installed' occurred while executing PIP install-r requires.txt. Then I tried to install nose, which is not installed'. SourceFileLoader has no attribute SourceFileLoader ', could you please give me some solutions? Thank you very much

**Comment by kungkook** [comment]
Hello, I was trying to run the code under ubuntu and 'torch encoding 1.0.1 required nose, which is not installed' occurred while executing PIP install-r requires.txt. Then I tried to install nose, which is not installed'. SourceFileLoader has no attribute SourceFileLoader ', could you please give me some solutions? Thank you very much
